data_loading.py

```diff
@@ -335,11 +335,19 @@ def get_variables():
 
     VARDICT["NUEVARS"] = NUEVARS
 
-    NUMUVARS = [
-        # "trk_energy_proton_v"
+    NUMUVARS = []
+    NUMU_TKI_VARS = [
+      "mc_pdg",
+      "mc_px",
```

Comment:
These rugged arrays for example will be loaded directly to the dataframe, it would be better to load them inside the processing functions to temporary variables instead. They will blow up the size of the DataFrames for the numu sideband.

Reply:
They're not loaded by default.

 I'll look at this in  the new year. Please merge this as the updated data loading is quite important for getting the note finished on time.

Reply:
ok merged